src/components/Footer/Footer.css

```diff
@@ -119,9 +119,9 @@ form {
     width: 100%;
     margin-top: 2rem;
     padding: 3%;
-    background: #6a00bb;
-    border-bottom: 2px solid #6a00bb;
-    border-radius: 2px;
+    background: #bb86fc;
```

Comment:
Declare variables for colors please with appropriate names

Reply:
okay